### PR TITLE
import & export configuration

### PR DESCRIPTION
**Pull request by tetuaoro**
## Description
This pull request adds import and export configuration features to `wg-easy`. Users can now easily save and load their configurations from the user interface.

## Changes
- Added import configuration feature.
- Added export configuration feature.
- Updated the user interface to include new import/export buttons.

Thank you for taking the time to review these changes ! Let me know if you have any feedback.

### CODE REVIEW

**Comment by pheiduck** [comment]
Other things looks fine.
The new function must be translated into the other languages we support.

**Comment by tetuaoro** [comment]
Hi,

I've added translations for French, Spanish, and Italian, and updated the terminology to use "restore" instead of "import" and "backup" instead of "export" for better clarity and relevance. Additionally, I renamed the relevant functions and updated the README with a new screenshot to reflect these changes.

screenshot

**Comment by pheiduck** [comment]
@tetuaoro as a bonus I added german translation to it.

**Review by pheiduck — APPROVED** [review]

File: src/lib/Server.js [hunk 1]
```diff
@@ -276,7 +293,7 @@ module.exports = class Server {
           getMeta: async (id) => {
             const filePath = safePathJoin(publicDir, id);
 
-            const stats = await stat(filePath).catch(() => {});
+            const stats = await stat(filePath).catch(() => { });
```
[the remark this answers is not in the document]
Comment (tetuaoro) [reply src/lib/Server.js:296]: Ah, maybe it's the `formatter` vscode extension or the `eslint`, idk.

**Comment by pheiduck** [comment]
Login is not possible with that, needs rework

**Comment by tetuaoro** [comment]
> Login is not possible with that, needs rework

How can I test that ? I use `--env PASSWORD=...` and I can login & logout.

**Comment by pheiduck** [comment]
reopen the PR, then we can make a development image first

**Pull request merged**